### PR TITLE
orting Changes forDeployAzureRG Task [U3 important bug fixes]

### PR DESCRIPTION
**Pull request by anaggar**

### CODE REVIEW

File: Tasks/DeployAzureResourceGroup/AzureUtilityLTE9.8.ps1 [hunk 1]
```diff
@@ -32,23 +32,26 @@ function Create-AzureResourceGroupIfNotExist
...
+        $finalCommand = "`$azureResourceGroupDeployment = New-AzureResourceGroupDeployment -Name `"$deploymentName`" -ResourceGroupName `"$resourceGroupName`" -TemplateFile `"$csmFile`" $overrideParameters -Verbose -ErrorAction silentlycontinue -ErrorVariable deploymentError"
     }
     else
     {
-        $finalCommand = "`$azureResourceGroupDeployment = New-AzureResourceGroupDeployment -Name `"$resourceGroupName`" -ResourceGroupName `"$resourceGroupName`" -TemplateFile `"$csmFile`" -TemplateParameterObject `$csmParametersObject $overrideParameters -Verbose -ErrorAction silentlycontinue -ErrorVariable deploymentError"
+        $finalCommand = "`$azureResourceGroupDeployment = New-AzureResourceGroupDeployment -Name `"$deploymentName`" -ResourceGroupName `"$resourceGroupName`" -TemplateFile `"$csmFile`" -TemplateParameterFile `$csmParametersFile $overrideParameters -Verbose -ErrorAction silentlycontinue -ErrorVariable deploymentError"
     }
 
-    Write-Verbose -Verbose "$finalCommand"
+    Write-Verbose "$finalCommand"
```
Comment (pavanadepu2) [inline Tasks/DeployAzureResourceGroup/AzureUtilityLTE9.8.ps1]: Ad verbose here.

Comment (pavanadepu2) [inline Tasks/DeployAzureResourceGroup/AzureUtilityLTE9.8.ps1]: Add verbose flag here.

File: Tasks/DeployAzureResourceGroup/Utility.ps1 [hunk 2]
```diff
@@ -798,47 +814,48 @@ function Validate-CustomScriptExecutionStatus
...
-        if($subStatuses)
-        {
-            foreach($subStatus in $subStatuses)
+            if($subStatuses)
             {
-                if($subStatus.Code.Contains("ComponentStatus/StdErr") -and (-not [string]::IsNullOrEmpty($subStatus.Message)))
+                foreach($subStatus in $subStatuses)
                 {
-                    $isScriptExecutionPassed = $false
-                    $errMessage = $subStatus.Message
```
Comment (pavanadepu2) [inline Tasks/DeployAzureResourceGroup/Utility.ps1]: I don't see any change here.

File: Tasks/Xcode/xcodebuild3.js [hunk 3]
```diff
@@ -2,6 +2,7 @@
   Copyright (c) Microsoft. All rights reserved.
```
Comment (pavanadepu2) [inline Tasks/Xcode/xcodebuild3.js]: This file is not related to our port changes. Please remove it.

**Comment by KrishnaAdityaB** [comment]
:shipit:
